### DISCUSSION TITLE
Disable Tracing via Environment Variable or Run Flag?

### DISCUSSION THREAD

**Discussion by leefernandes** [open-policy-agent/community#508] [Envoy] [answered]
Does anyone know if it's possible to enable/disable distributed tracing with environment variables or command line flags to `opa run`?

The only option I see for enabling distributed tracing is via the configuration file.  It'd be easier with our cicd configuration to provide an option via environment variable or `run` command line argument.

**Comment by srenatus** [comment, marked as answer]
You should be able to override any config file setting using `--set`, like
```
opa run --server --config-file config.yml --set distributed_tracing=null
```

**Reply by anderseknert** [reply]
Yep, see https://www.openpolicyagent.org/docs/latest/configuration/#cli-runtime-overrides
